test/unittest/blas3/blas3_gemm_batched_test.cpp

```diff
@@ -146,6 +146,27 @@ const auto AllStridedBatched =
     );
 GENERATE_GEMM_STRIDED_BATCHED_TEST(BatchStridedGemm, AllStridedBatched);
 
+template <typename scalar_t>
+const auto ZerosStridedBatched =
```

Comment:
I think we can merge this test combination with the `AllStridedBatched` one.

Reply:
Yes will add it ;)